### PR TITLE
improve reasoning for SD-JWT

### PR DESCRIPTION
**Pull request by paulbastian**
See #94

### CODE REVIEW

**Review by tlodderstedt — APPROVED** [review]

File: draft-terbu-sd-jwt-vc.md [hunk 1]
```diff
@@ -93,10 +93,10 @@ two approaches is provided in this specification.
 
 JSON Web Tokens (JWTs) [@!RFC7519] can in principle be used to express
 Verifiable Credentials in a way that is easy to understand and process as it
-builds upon established web primitives. However, JWT-based credentials do not
-support selective disclosure, i.e., the ability for a Holder to disclose only a
-subset of the claims contained in the JWT, which is a requirement to implement
-the three-party-model efficiently.
+builds upon established web primitives. While JWT-based credentials enable selective
+disclosure, i.e., the ability for a holder to disclose only a subset of the contained
```
Comment (peppelinux) [inline draft-terbu-sd-jwt-vc.md:97]: holder -> Holder

defined term

**Review by peppelinux — CHANGES_REQUESTED** [review]

**Review by awoie — APPROVED** [review]
lgtm

**Comment by awoie** [comment]
Merging, two approvals.

**Comment by awoie** [comment]
fixed #94

**Pull request merged**